### PR TITLE
Investigate name change

### PR DESCRIPTION
**Pull request by StRhys**

### CODE REVIEW

File: volto/src/addons/volto-chart-builder/src/components/ChartBuilderEdit.jsx [hunk 1]
```diff
@@ -18,9 +18,17 @@ const View = () => {
...
-  const { selected, onChangeBlock, block, data } = props;
-  const { error: dataError } = usePloneCsvData(data.file_path || []);
-  const { error: geoJsonError } = usePloneGeoJson(data.geojson_path || []);
+  const { selected, onChangeBlock, block, data, properties } = props;
+
+  const { error: dataError } = usePloneCsvData(
+    properties.parent,
+    data.file_path || [],
+  );
+  // this error does not seem to be being set, despite there being a problem in usePloneCsvData
```
Comment (mikeAdamss) [inline volto/src/addons/volto-chart-builder/src/components/ChartBuilderEdit.jsx:27]: can we address this or make sure there's a card too do so please.

**Review by mikeAdamss — APPROVED** [review]
from what I can understand it looks good to me 👍 , approved and very minor comments only.

one wider point - annotating function and methods with a few lines of "what this does is.... " type comments is very important, it seems pointless now while you live and breathe volto, but  when your responsibilities (or the codebase) grow you'll kick yourself hard for not doing more.

I don't mean now, or some sort of mega docstring sweep, but every time you're in the codebase making a change, add a few lines of explanation where they're missing and where you're certain of whats going on - future you will thank you 😄

File: volto/src/helpers/index.js [hunk 2]
```diff
@@ -4,3 +4,17 @@
  * @example
  * export { Api } from './Api/Api';
  */
+
+const replaceUrl = (originalPath, parentPath) => {
```
[the remark this answers is not in the document]
Comment (StRhys) [reply volto/src/helpers/index.js:17]: Added some comments explaining function

File: volto/src/addons/volto-chart-builder/src/components/ChartBuilderEdit.jsx (hunk 1 above)
Comment (StRhys) [reply volto/src/addons/volto-chart-builder/src/components/ChartBuilderEdit.jsx:27]: Made a ticket to investigate this.
https://app.zenhub.com/workspaces/cogs-dd-cms-61e83b63053eb70011d37b4f/issues/gh/gss-cogs/dd-cms/844

**Pull request merged**